### ISSUE TITLE
Vendor can't add a product picture

### ISSUE THREAD

**Issue by pacino01** [eckmarcommunity/eckmar#91]
![vendor](https://github.com/eckmarcommunity/eckmar/assets/152064237/df3a0af9-56e9-45b3-88af-fc57b3e48835)

**Comment by ffeel31** [comment]
i have same king of issue

![Uploading FireShot Capture 005 - Colombiano - Products - 192.168.100.135.png…]()

**Comment by ffeel31** [comment]
I have same kind of issue. any picture i try to upload it says "the Picture must be and Image"
please help
 ![FireShot Capture 005 ](https://github.com/eckmarcommunity/eckmar/assets/152450002/5449f07c-54f8-42df-ba07-2609877685bb)

**Comment by knulii** [comment]
try a .png image

**Comment by ffeel31** [comment]
but i want to be able to upload other picture formats like jpg and gif. 
I have tried to modify the code here like this but still not working

public function rules()
    {
        return [
            'picture' => 'required|image| mimes:jpeg, png, jpg, gif | max: 1024 | dimensions:min_width=256,min_height=256',
            'first' => 'boolean|nullable',
        ];
    }

**Comment by pacino01** [comment]
> Sounds like a permission issue with the storage/public folder double check you got the permissions correct https://github.com/eckmarcommunity/eckmar/blob/master/INSTALLATION.md#permissions

That is correct. Just execute the permissions again as a root user. It must be a PNG File.

**Comment by pacino01** [comment]
> but i want to be able to upload other picture formats like jpg and gif. I have tried to modify the code here like this but still not working
> 
> public function rules() { return [ 'picture' => 'required|image| mimes:jpeg, png, jpg, gif | max: 1024 | dimensions:min_width=256,min_height=256', 'first' => 'boolean|nullable', ]; }

Did you resolved your issue? what was your solution to extend the picture uploading?

**Comment by Gralien** [comment]
Type of file. And size.
Check this.
